Code/ElastixTransformixWrappers/src/sitkTransformixImageFilterImpl.cxx

```diff
@@ -125,7 +126,7 @@ TransformixImageFilter::TransformixImageFilterImpl ::TransformixImageFilterImpl(
   this->m_MemberFactory->RegisterMemberFunctions<FloatPixelIDTypeList, 2>();
   this->m_MemberFactory->RegisterMemberFunctions<FloatPixelIDTypeList, 3>();
 
-#if SITK_MAX_DIMENSIONS >= 4
+#if SITK_MAX_DIMENSION >= 4
```

Comment:
- Looks like this bug was introduced with pull request #1892 right?

Anyway, "sitkConfigure.h.in" suggests `SITK_MAX_DIMENSION` without `S`, so that should be right  👍 
https://github.com/SimpleITK/SimpleITK/blob/2c9389c0b570cb40ddea7e7ab26f41846e475fe9/Code/Common/src/sitkConfigure.h.in#L30

Reply:
I am not sure 4D was working before that change either.